### ISSUE TITLE
Throwing NullPointerException while getting attachments

### ISSUE THREAD

**Issue by jagadeesh1035** [shivam1608/JMailTM#9]
I could see you have fixed it but it is not available in 0.7.0

**Comment by shivam1608** [comment]
@jagadeesh1035 it will be present in next release till then you can use version as `main-SNAPSHOT` to use the latest version.

```
<dependency>
	    <groupId>com.github.shivam1608</groupId>
	    <artifactId>JMailTM</artifactId>
	    <version>main-SNAPSHOT</version>
</dependency>
```

**Comment by shivam1608** [comment]
issue fixed in release `0.7.1`

closing the issue.
